### DISCUSSION TITLE
Hi, configuration file env.local When was it loaded into the application?

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#719] [Q&A] [answered]
Hi, configuration file env.local When was it loaded into the application? I want to control where it is placed, but I can't find the code to load it.

**Comment by calebmshafer** [comment, marked as answer]
Hey @shenshengyi, if you're using react-scripts (`@bentley/react-scripts`) you can go over to the [create-react-app](https://create-react-app.dev/docs/adding-custom-environment-variables#what-other-env-files-can-be-used) for more details on the way they load the files.  All of the `.env` files are expected to be located as a peer of the `package.json`.

**Reply by shenshengyi** [reply]
> Hey @shenshengyi, if you're using react-scripts (`@bentley/react-scripts`) you can go over to the [create-react-app](https://create-react-app.dev/docs/adding-custom-environment-variables#what-other-env-files-can-be-used) for more details on the way they load the files. All of the `.env` files are expected to be located as a peer of the `package.json`.



OK, thank you very much.
